ci/deps/travis-36-locale.yaml

```diff
@@ -35,7 +35,7 @@ dependencies:
   - xlwt
   # universal
   - pytest>=4.0.2
```

Comment:
Not sure if we need this min here either but wasn't touched earlier so leaving as is